src/lhttpc_client.erl

```diff
@@ -972,3 +973,14 @@ fix_inet_options(Options) ->
                 end;
             (Option) -> {true, Option}
         end, Options).
+
+-ifdef(cacerts).
+add_cacerts(ConnOpts) ->
+    case proplists:is_defined(cacerts, ConnOpts) of
+        true -> ConnOpts;
+        false ->
+            [{cacerts, public_key:cacerts_get()} | ConnOpts]
+    end.
```

Comment:
I think it would be even better to provide the client with an option to avoid calling the `public_key:cacerts get()`. So it can be a workarounded in cases it causes any issues.